seed.js

```diff
@@ -1,9 +1,34 @@
-const mongoose = require('mongoose');
+const faker = require('faker');
 
-mongoose.connect('mongodb://localhost/listings/:listing_id/recommendations');
+const genRecommendation = (amount) => {
+  const recommendations = [];
+  for (let i = 0; i < amount; i += 1) {
+    const recommendation = {
+      title: faker.address.city(),
+      description: faker.lorem.sentences(),
+      price: faker.commerce.price(),
+      photo_url: faker.image.imageUrl(),
```

Comment:
This generates the same url every time, but the url displays a different image every time you visit - I just mention to be sure that's what you want!

Reply:
I'll look into generating unique images when I'm refactoring. Thanks Eric!